docs/source/torch.rst

```diff
@@ -260,11 +260,7 @@ Examples::
     :toctree: generated
     :nosignatures:
 
-    no_grad
-    enable_grad
-    set_grad_enabled
```

Comment:
I think the text just above does refer to these functions. Maybe removing the ones in `autograd.rst` would be less disruptive? We can also add plain text there that links to the `torch.` version of these docs.

Reply:
That's what I originally did. But I like @NicolasHug's comment:

> All of them are already mentioned and clickable [just above](https://github.com/pytorch/pytorch/blob/master/docs/source/torch.rst#locally-disabling-gradient-computation), except for `torch.is_grad_enabled`, but we can add it as well.

So the idea is that the table remains in `autograd.rst` but `torch.rst` still has links to the same docs in approximately the same location.

I'm happy to remove whichever table entries you decide on though.

Reply:
> All of them are already mentioned and clickable [just above](https://github.com/pytorch/pytorch/blob/master/docs/source/torch.rst#locally-disabling-gradient-computation), except for torch.is_grad_enabled, but we can add it as well.

But they are clickable only because we still have entries for `torch.` versions. If we only keep the `torch.autograd.` versions, these won't be clickable anymore right?

Reply:
The `torch.no_grad()` usage is definitely more common. That's why I was suggesting removing the `autograd.rst` table. We can just replace that table with a link to the `torch.rst` paragraph that explain these functions.

Reply:
Done